lib/routes/account.js

```diff
@@ -1853,7 +1934,7 @@ module.exports = function (
                       }
 
                       // Any matching code verifies the account
-                      return db.verifyEmail(account)
+                      return db.verifyEmail(account, code)
```

Comment:
IIUC, this will break the functionality where signin verification also verifies the primary account email:

* Create an account, but don't click through the initial verification email.
* Sign in to that account in a different browser.  You'll get a fresh email telling you to verify the account, with the tokenVerificationId for the new session instead of the emailCode from the account.
* Click that that second email.  The code will be accepted and the session verified due to logic above.
* But on this one, `code` is the tokenVerificationId and is not equal to `account.emailCode`.  The db-server will look for emails matching that code, fail to find any, and leave the account in the unverified state.

We *should* have a test-case that catches this, can you please check whether we do?  (And/or, whether my logic above is incorrect?).